### PR TITLE
[T3A2][W11-A1]Tay Keming Justin

### PR DESCRIPTION
**Pull request by INCENDE**

### CODE REVIEW

**Comment by INCENDE** [comment]
ready for review

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if(resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
     }
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: good catch

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -10,27 +10,34 @@
...
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
-        return Optional.ofNullable(relevantPersons);
+	return Optional.ofNullable(relevantPersons);
+    }
+
+    public String getFeedbackToUser() {
+	return feedbackToUser;
     }
 
 }
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: Looks like this time you fixed all the formatting to adhere to the coding style. nice!

**Comment by cmkumar87** [comment]
@INCENDE Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Pull request closed without merging**